nmigen_boards/ecp5_5g_evn.py

```diff
@@ -15,20 +15,117 @@ class ECP55GEVNPlatform(LatticeECP5Platform):
     speed       = "8"
     default_clk = "clk12"
     default_rst = "rst"
+    def __init__(self, *, VCCIO1="LVCMOS25", VCCIO2="LVCMOS33", VCCIO6="LVCMOS33", VCCIO7="LVCMOS33", **kwargs):
+        assert VCCIO1 in ["LVCMOS33", "LVCMOS25"]
+        assert VCCIO2 in ["LVCMOS33", "LVCMOS25"]
+        assert VCCIO6 in ["LVCMOS33", "LVCMOS25"]
+        assert VCCIO7 in ["LVCMOS33", "LVCMOS25", "LVCMOS15", "LVCMOS12"]
+        self._VCCIO1 = VCCIO1
+        self._VCCIO2 = VCCIO2
+        self._VCCIO6 = VCCIO6
+        self._VCCIO7 = VCCIO7
+    
+    def VCCIO1(self):
```

Comment:
These functions really return an IO standard, not a voltage, so they should be called something like `bank1_iostandard` like in the Atlys board.